README.md

```diff
@@ -42,6 +42,80 @@ registerBreadcrumbHelper()
 res.render(breadcrumbsPath: [{url: 'https://kth.se', label: 'KTH'}, ...], ...)
 ```
 
+## Language Link Helper
+
+Handlebars helper to generate a language link in the header. If language can’t be toggled with query parameter `l`, then there is an option to display a dialog with a custom link.
+
+### Register helper
+
+```javascript
+// Typically server/views/helpers/index.js
+
+const { registerLanguageLinkHelper } = require('@kth/kth-node-web-common/lib/handlebars/helpers/languageLink')
+
+registerLanguageLinkHelper()
+```
+
+### Add language constants
+
+All language constants are optional.
+
+- `label_lang_[en/sv]`: Default label for the anchor element's text, if a custom one isn’t provided. Remember that it should be displayed in the opposite language, e.g. if the page is in English the label should be _Svenska_
+- `label_button_close`: The label for the close button in the dialog element. Only used if there’s a dialog.
+- `label_not_translated`: The label for the dialog element's text. Only used if there’s a dialog.
+
+```javascript
+// Example in i18n/messages.se.js
+
+label_lang_en: 'English',
+label_not_translated: 'Den här sidan är ej översatt',
+label_button_close: 'Stäng',
+```
+
+### Styling
+
+Make sure to include styling from KTH Style.
+
+```css
+/* Typically application’s main Sass-file */
+
+@use '~@kth/style/scss/components/translation-panel';
+```
+
+### Initialize menu panel
+
+Make sure to initialize the menu panel (dialog). _This might be moved to KTH Style._
+
+```javascript
+// Typically in server/views/layouts/publicLayout.handlebars
+
+<script type="module">
+  import {MenuPanel} from '{{ proxyPrefix }}/assets/js/index.js' MenuPanel.initTranslationModal(
+  document.querySelector(".kth-menu-item.language"), document.querySelector(".kth-translation") )
+</script>
+```
+
+### Use handlebars helper in head
+
+Include the handlebars helper in the template.
+
+```handlebars
+<!-- Typically in server/views/partials/kthHeader.handlebars -->
+
+{{{languageLink lang}}}
```

Comment:
Is this where you "configure" the helper, passing "link", "dialogMessage" etc?

How does a more complex implementation look?

Reply:
Added documentation for a more complex implementation.